### ISSUE TITLE
allthemodium-2.8.4_mc_1.21.1 neoforge issue

### ISSUE THREAD

**Issue by mistercapitale** [AllTheMods/AllTheModium#137]
Hello,
I'm not sure if you are aware of this issue or not, but I have updated my pack with the latest version of allthemodium. When I try and use any of the pickaxes including vanilla, it wont break anything the way that they are intended. Let me know if you need anything more than that to help you. Thank you. Appreciate all your work on this mod.

**Comment by ghost** [comment]
I also encountered this problem

**Comment by mistercapitale** [comment]
Hello, I was just checking in to see if there has been any movement on this? I see that there are updates, but the issue is still there with every update...

**Comment by thevortex** [comment]
are you saying pickaxes dont work?

**Comment by mistercapitale** [comment]
Yep, I tested a majority of the pickaxes and it won’t break stone or anything. I turn allthemodium off and I get the use out of pickaxes again. I even started a new world with just that mod and it still doesn't work.

**Comment by thevortex** [comment]
cannot reproduce on current version sorry wooden, stone, iron pickaxes all function as expected on ATM10 with current version of Allthemodium

**Comment by mistercapitale** [comment]
Did you try with a fresh 1.21.1 neoforge with just allthemodium? Because I got the same results with that too.

I can show you the setup and post a video if you'd like to see it?

**Comment by mistercapitale** [comment]
Here is what ChatGPT said about my log file: Your log contains the following issues:

Reference Warnings:
geckolib.refmap.json could not be read. If you're in a development environment, you can ignore this, but it may indicate an incomplete or misconfigured mod installation.
Missing Resources:
minecraft:models/item/handshield.json is missing, referenced by the Vibranium Shield model.
Missing sound events for goat horn-related sounds.
Missing Tags and References:
Tags like minecraft:mineable/pickaxe and allthemodium:paxel_effective are missing required references. These seem to involve custom mod items or ores.
Loot Table and Recipe Errors:
The loot table allthemodium:blocks/allthemodium_source_jar cannot parse due to missing or unknown registry keys.
Recipe errors for alltheores crushing recipes like iridium, osmium, lead, diamond, and tin. These errors reference an unknown serializer actuallyadditions:crushing, indicating either a missing dependency or outdated configuration.

[latest.log](https://github.com/user-attachments/files/18483346/latest.log)

**Comment by thevortex** [comment]
just tested in a fresh install, no issues with tools

**Comment by mistercapitale** [comment]
So with my fresh install it's talking about the tags mineable/pickaxe are missing required references, that would make things not mineable correct? I just don't understand how myself and someone else are having issues with this on a fresh install and you aren't. I'm not sure where to go from here.

**Comment by mistercapitale** [comment]
I tried one more thing, I downloaded Allthemods 10 and it worked fine, however when I deleted all of the mods accept allthemodium and gecko, then again im not able to break stone. So somewhere a setting or mod puts it back to working.

**Comment by thevortex** [comment]
i'll look into this more tomorrow, i have a feeling a recent feature addition for ars wasnt done correctly, will update tomorrow

**Comment by mistercapitale** [comment]
Thank you so much, appreciate it thevortex.

**Comment by mistercapitale** [comment]
@thevortex any luck? asking for a friend.:)

**Comment by thevortex** [comment]
right now if you want an easy fix, you can add a one liner in kubejs to remove the allthemodium source jar from the minecraft:mineable/pickaxe tag, i am still trying to find a workaround that gets this tag in place when it is in fact needed

**Comment by mistercapitale** [comment]
@thevortex I'm going to be real honest, I'm as dumb as door knob when it comes to programing. Ive done all my KubeJS scripting via ChatGPT and that within itself is a struggle bus. I'm not exactly sure how to do what you just said. lol. Sorry.

**Comment by mistercapitale** [comment]
T H A N K  Y O U ! ! ! !

**Comment by thevortex** [comment]
confirmed good then @mistercapitale ?

**Comment by mistercapitale** [comment]
@thevortex Yep, confirmed working again. Thanks for all your help. I just couldn't release my modpack I made without this mod. It's one of my all time favorites. Appreciate you taking the time.
